src/commands/snapshot.ts

```diff
@@ -0,0 +1,93 @@
+import {flags} from '@oclif/command'
+import Constants from '../services/constants'
+import {StaticSnapshotOptions} from '../services/static-snapshot-options'
+import StaticSnapshotService from '../services/static-snapshot-service'
+import PercyCommand from './percy-command'
+
+export default class Snapshot extends PercyCommand {
+  static description = 'Snapshot a directory of webpages'
+  static hidden = true
+
+  static args = [{
+    name: 'snapshotDirectory',
+    description: 'A path to the directory you would like to snapshot',
+    required: true,
+  }]
+
+  static examples = [
+    '$ percy snapshot _site/',
+    '$ percy snapshot _site/ --baseUrl "blog/"',
```

Comment:
The example here is now out of date:

```suggestion
    '$ percy snapshot _site/ --base-url "blog/"',
```